build.gradle

```diff
@@ -190,7 +190,7 @@ dependencies {
     compile 'com.amazon.opendistro.elasticsearch:performanceanalyzer-rca:1.10'
     compile 'com.fasterxml.jackson.core:jackson-annotations:2.8.11'
     compile 'com.fasterxml.jackson.core:jackson-databind:2.8.11'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.8.11'
+    compile 'com.fasterxml.jackson.module:jackson-module-paranamer:2.11.3'
```

Comment:
We might have to check if this doesn't cause any regression in AES.

Reply:
Good call out, any way to quickly verify it? Also, this module is supposed to be used in testing ONLY.